ConsoleApp1/Helpers.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    class Helpers
+    {
+        //public static void DeleteWrongCars()
+        //{
+        //    CrmServiceClient crmSvc = new CrmServiceClient(ConfigurationManager.ConnectionStrings["MyCRMServer"].ConnectionString);
+
+        //    string query = @"
+        //    <fetch version='1.0' output-format='xml-platform' mapping='logical' distinct='false'>
+        //        <entity name='new_car'>
+        //        <attribute name='new_vin' />
+        //        <attribute name='new_purchase' />
+        //        <attribute name='new_productiondate' />
+        //        <attribute name='new_carmodel' />
+        //        <attribute name='new_carmanufacturer' />
+        //        <attribute name='new_carclass' />
+        //        <attribute name='new_carid' />
+        //        <order attribute='new_vin' descending='false' />
+        //        <filter type='and'>
+        //            <condition attribute='new_productiondate' operator='gt' valueof='new_purchase'/>
+        //        </filter>
+        //        </entity>
+        //    </fetch>";
+
+        //    EntityCollection entityCollection = crmSvc.RetrieveMultiple(new FetchExpression(query));
+
+        //    int i = 0;
+        //    foreach (Entity car in entityCollection.Entities)
+        //    {
+        //        i++;
+        //        //crmSvc.Delete("new_car", new Guid(car.Attributes["new_carid"].ToString()));
+        //        Console.WriteLine(car.Attributes["new_vin"].ToString());
+        //        Console.WriteLine(car.Attributes["new_purchase"].ToString());
+        //        Console.WriteLine(car.Attributes["new_productiondate"].ToString());
+        //    }
+        //    Console.WriteLine(i);
+        //    Console.Read();
+        //} // TODO: to delete?
+
+        public static DateTime GenerateDateInRange(DateTime startDate, DateTime endDate, Random rnd)
+        {
+            TimeSpan timeSpan = endDate - startDate;
+            TimeSpan newSpan = new TimeSpan(0, rnd.Next(0, (int)timeSpan.TotalMinutes), 0);
```

Comment:
what does newSpan variable name mean ?

Reply:
Just a TimeSpan with random duration in between the given dates.